athletics.lic

```diff
@@ -353,9 +358,23 @@ class Athletics
       walk_to(6443)
       break if done_training?
       walk_to(@outdoorsmanship_rooms.sample) unless @outdoorsmanship_rooms.empty?
-      train_outdoorsmanship
-      train_outdoorsmanship
-      train_outdoorsmanship
+	  4.times do
```

Comment:
this should be 3.times, the travel to and from the area takes over 15 seconds thats why its shorter

Reply:
Okay